.xperior/testds/motr-single_tests.yaml

```diff
@@ -19,8 +19,1268 @@
 
 ---
 Tests:
-  - id       : 00userspace-tests
-    script   : 'm0 run-ut'
+  - id       : 00userspace-tests_libm0-ut
```

Comment:
I suggest to have a new script to run all the UTs in it. Something like this:
```
[520428@ssc-vm-g3-rhev4-2904 cortx-motr]$ cat run_ut.sh
timeout 2400 ./scripts/m0 run-ut -t libm0-ut
timeout 2400 ./scripts/m0 run-ut -t addb2-base
timeout 2400 ./scripts/m0 run-ut -t addb2-consumer
timeout 2400 ./scripts/m0 run-ut -t addb2-histogram
timeout 2400 ./scripts/m0 run-ut -t addb2-net
timeout 2400 ./scripts/m0 run-ut -t addb2-storage
timeout 2400 ./scripts/m0 run-ut -t addb2-sys
timeout 2400 ./scripts/m0 run-ut -t di-ut
timeout 2400 ./scripts/m0 run-ut -t balloc-ut
timeout 2400 ./scripts/m0 run-ut -t be-ut
timeout 2400 ./scripts/m0 run-ut -t buffer_pool_ut
timeout 2400 ./scripts/m0 run-ut -t bulk-client-ut
timeout 2400 ./scripts/m0 run-ut -t bulk-server-ut
timeout 2400 ./scripts/m0 run-ut -t bytecount-ut
timeout 2400 ./scripts/m0 run-ut -t dtm0-ut
timeout 2400 ./scripts/m0 run-ut -t btree-ut
timeout 2400 ./scripts/m0 run-ut -t capa-ut
timeout 2400 ./scripts/m0 run-ut -t cas-client
timeout 2400 ./scripts/m0 run-ut -t cas-service
timeout 2400 ./scripts/m0 run-ut -t client-ut
timeout 2400 ./scripts/m0 run-ut -t obj-ut
timeout 2400 ./scripts/m0 run-ut -t io-ut
timeout 2400 ./scripts/m0 run-ut -t io-nw-xfer-ut
timeout 2400 ./scripts/m0 run-ut -t io-pargrp-ut
timeout 2400 ./scripts/m0 run-ut -t io-req-ut
timeout 2400 ./scripts/m0 run-ut -t io-req-fop-ut
timeout 2400 ./scripts/m0 run-ut -t sync-ut
timeout 2400 ./scripts/m0 run-ut -t idx-ut
timeout 2400 ./scripts/m0 run-ut -t idx-dix
timeout 2400 ./scripts/m0 run-ut -t idx-dix-mt
timeout 2400 ./scripts/m0 run-ut -t layout-ut
timeout 2400 ./scripts/m0 run-ut -t helpers-ufid-ut
timeout 2400 ./scripts/m0 run-ut -t cm-cp-ut
timeout 2400 ./scripts/m0 run-ut -t cm-ut
timeout 2400 ./scripts/m0 run-ut -t cob-ut
timeout 2400 ./scripts/m0 run-ut -t cob-foms-ut
timeout 2400 ./scripts/m0 run-ut -t conf-ut
timeout 2400 ./scripts/m0 run-ut -t conf-load-ut
timeout 2400 ./scripts/m0 run-ut -t conf-pvers-ut
timeout 2400 ./scripts/m0 run-ut -t confc-ut
timeout 2400 ./scripts/m0 run-ut -t conf-glob-ut
timeout 2400 ./scripts/m0 run-ut -t conf-diter-ut
timeout 2400 ./scripts/m0 run-ut -t confstr-ut
timeout 2400 ./scripts/m0 run-ut -t conf-validation-ut
timeout 2400 ./scripts/m0 run-ut -t conf-walk-ut
timeout 2400 ./scripts/m0 run-ut -t rconfc-ut
timeout 2400 ./scripts/m0 run-ut -t rpc-connection-ut
timeout 2400 ./scripts/m0 run-ut -t dix-client-ut
timeout 2400 ./scripts/m0 run-ut -t dix-cm-iter
timeout 2400 ./scripts/m0 run-ut -t dtm-nucleus-ut
timeout 2400 ./scripts/m0 run-ut -t dtm-transmit-ut
timeout 2400 ./scripts/m0 run-ut -t dtm-dtx-ut
timeout 2400 ./scripts/m0 run-ut -t dtm0-clk-src-ut
timeout 2400 ./scripts/m0 run-ut -t dtm0-log-ut
timeout 2400 ./scripts/m0 run-ut -t failure_domains_tree-ut
timeout 2400 ./scripts/m0 run-ut -t failure_domains-ut
timeout 2400 ./scripts/m0 run-ut -t fis-ut
timeout 2400 ./scripts/m0 run-ut -t fdmi-filterc-ut
timeout 2400 ./scripts/m0 run-ut -t fdmi-pd-ut
timeout 2400 ./scripts/m0 run-ut -t fdmi-sd-ut
timeout 2400 ./scripts/m0 run-ut -t fdmi-fol-ut
timeout 2400 ./scripts/m0 run-ut -t fdmi-fol-fini-ut
timeout 2400 ./scripts/m0 run-ut -t fdmi-filter-eval-ut
timeout 2400 ./scripts/m0 run-ut -t fit-ut
timeout 2400 ./scripts/m0 run-ut -t fol-ut
timeout 2400 ./scripts/m0 run-ut -t fom-timedwait-ut
timeout 2400 ./scripts/m0 run-ut -t rpc-formation-ut
timeout 2400 ./scripts/m0 run-ut -t ha-ut
timeout 2400 ./scripts/m0 run-ut -t ha-state-ut
timeout 2400 ./scripts/m0 run-ut -t ios-bufferpool-ut
timeout 2400 ./scripts/m0 run-ut -t isc-api-ut
timeout 2400 ./scripts/m0 run-ut -t isc-service-ut
timeout 2400 ./scripts/m0 run-ut -t rpc-item-ut
timeout 2400 ./scripts/m0 run-ut -t rpc-item-source-ut
timeout 2400 ./scripts/m0 run-ut -t layout-ut
timeout 2400 ./scripts/m0 run-ut -t layout-access-plan-ut
timeout 2400 ./scripts/m0 run-ut -t rpc-link-ut
timeout 2400 ./scripts/m0 run-ut -t fop-lock-ut
timeout 2400 ./scripts/m0 run-ut -t fom-stats-ut
timeout 2400 ./scripts/m0 run-ut -t net-bulk-if
timeout 2400 ./scripts/m0 run-ut -t net-bulk-mem
timeout 2400 ./scripts/m0 run-ut -t net-lnet
timeout 2400 ./scripts/m0 run-ut -t libfab-ut
timeout 2400 ./scripts/m0 run-ut -t net-misc-ut
timeout 2400 ./scripts/m0 run-ut -t net-module
timeout 2400 ./scripts/m0 run-ut -t net-test
timeout 2400 ./scripts/m0 run-ut -t net-prov-ut
timeout 2400 ./scripts/m0 run-ut -t m0d-ut
timeout 2400 ./scripts/m0 run-ut -t mdservice-ut
timeout 2400 ./scripts/m0 run-ut -t module-ut
timeout 2400 ./scripts/m0 run-ut -t ms-fom-ut
timeout 2400 ./scripts/m0 run-ut -t rpc-packet-encdec-ut
timeout 2400 ./scripts/m0 run-ut -t parity_math-ut
timeout 2400 ./scripts/m0 run-ut -t parity_math_ssse3-ut
timeout 2400 ./scripts/m0 run-ut -t poolmach-ut
timeout 2400 ./scripts/m0 run-ut -t reqh-ut
timeout 2400 ./scripts/m0 run-ut -t reqh-fop-allow-ut
timeout 2400 ./scripts/m0 run-ut -t reqh-service-ut
timeout 2400 ./scripts/m0 run-ut -t reqh-service-ctx-ut
timeout 2400 ./scripts/m0 run-ut -t rm-ut
timeout 2400 ./scripts/m0 run-ut -t rm-rcredits-ut
timeout 2400 ./scripts/m0 run-ut -t rm-rwlock-ut
timeout 2400 ./scripts/m0 run-ut -t rpc-at
timeout 2400 ./scripts/m0 run-ut -t rpc-machine-ut
timeout 2400 ./scripts/m0 run-ut -t rpc-rcv-session-ut
timeout 2400 ./scripts/m0 run-ut -t rpc-lib-ut
timeout 2400 ./scripts/m0 run-ut -t rpc-conn-pool-ut
timeout 2400 ./scripts/m0 run-ut -t rpc-session-ut
timeout 2400 ./scripts/m0 run-ut -t sm-ut
timeout 2400 ./scripts/m0 run-ut -t snscm_xform-ut
timeout 2400 ./scripts/m0 run-ut -t snscm_storage-ut
timeout 2400 ./scripts/m0 run-ut -t sns-cm-repair-ut
timeout 2400 ./scripts/m0 run-ut -t snscm_net-ut
timeout 2400 ./scripts/m0 run-ut -t sns-file-lock-ut
timeout 2400 ./scripts/m0 run-ut -t spiel-ut
timeout 2400 ./scripts/m0 run-ut -t spiel-ci-ut
timeout 2400 ./scripts/m0 run-ut -t sss-ut
timeout 2400 ./scripts/m0 run-ut -t stats-ut
timeout 2400 ./scripts/m0 run-ut -t spiel-conf-ut
timeout 2400 ./scripts/m0 run-ut -t stob-ut
timeout 2400 ./scripts/m0 run-ut -t storage-dev-ut
timeout 2400 ./scripts/m0 run-ut -t udb-ut
timeout 2400 ./scripts/m0 run-ut -t xcode_bufvec_fop-ut
timeout 2400 ./scripts/m0 run-ut -t ff2c-ut
timeout 2400 ./scripts/m0 run-ut -t xcode-ut
timeout 2400 ./scripts/m0 run-ut -t pi_ut
timeout 2400 ./scripts/m0 run-ut -t libconsole-ut

```

We will run this script as 00user-space.

Reply:
If we do the above then we will have to go and check the details of the test failure to figure out which particular test failed. Also this way the file is more readable as the particular snip in yaml is repeating and only the test name and id is changing. 
Besides that the current approach is working fine and below are the results.
![image](https://user-images.githubusercontent.com/44196748/187396855-6dfd510c-dad5-4198-ac18-84f8a2c7f81f.png)

Reply:
OK.